k6/tests/steve_watch_benchmark.js

```diff
@@ -0,0 +1,225 @@
+import { check, fail, sleep } from 'k6';
+import http from 'k6/http';
+import { Trend } from 'k6/metrics';
+import { WebSocket } from 'k6/experimental/websockets';
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
+// Parameters
+const steveServers = (__ENV.STEVE_SERVERS || 'http://localhost:8080').split(',');
+const namespace = __ENV.NAMESPACE || 'scalability-tests';
+const resource = __ENV.RESOURCE || 'configmaps';
+const changeRate = parseInt(__ENV.CHANGE_RATE || 1);
+const watchMode = __ENV.WATCH_MODE || ''; // "" for full resource, "resource.changes" for notifications
+const numConfigMaps = parseInt(__ENV.CONFIG_MAP_COUNT || 100);
+const vus = parseInt(__ENV.VUS || 1);
+const watchDuration = parseInt(__ENV.WATCH_DURATION || 30);
+
+const username = __ENV.USERNAME;
+const password = __ENV.PASSWORD;
+const token = __ENV.TOKEN;
+
+const setupTimeout = numConfigMaps / 10;
+const setupSettleTime = numConfigMaps * 0.01;
```

Comment:
nit: these achieve the same thing with different mathematical operations. Would be nice to standardize (either divide `setupSettleTime` by 100 or multiply `setupTimeout` by 0.1).

Also wondering if we should do some `min` operation here in case `numConfigMaps` is some very high number in the tens of thousands so that we aren't waiting for an arbitrarily long amount of time.